backend/tests/test_stub.py

```diff
@@ -54,21 +54,38 @@ async def test_books_add_endpoint(book_dict):
     assert response.status_code == 201
     assert response.headers.get("Content-Type") == "application/json"
 
-    book = await Book.objects.get(id=book_dict["id"])
-    assert str(book.id) == book_dict["id"]
-    assert book.counter == book_dict["counter"]
-    assert book.period == datetime.date.fromisoformat(book_dict["period"])
-    assert book.article_count == book_dict["article_count"]
-    assert book.size == book_dict["size"]
-    assert book.zimcheck == book_dict["zimcheck"]
+    _book = await Book.objects.get(id=book_dict["id"])
```

Comment:
python convention is to use `_` as a suffix when you don't want it to collide with another variable. so `book_` instead.